dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -178,7 +178,7 @@ export default defineComponent({
           ? kpiValue.map((naceCodeShort: string) => naceCodeMap.get(naceCodeShort)?.label ?? naceCodeShort)
```

Comment:
In line 140, we could wirte:
`for (const [kpiKey, kpiValue] of Object.entries(subCategoryObject) as [string, KpiValue][])`
instead of
`for (const [kpiKey, kpiValue] of Object.entries(subCategoryObject) as [string, object][])`

Reply:
Done.

Reply:
👍